consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/statemachine/IbftRoundFactory.java

```diff
@@ -73,8 +79,8 @@ public IbftRound createNewRoundWithState(
         protocolSchedule.getByBlockNumber(roundIdentifier.getSequenceNumber()).getBlockImporter(),
         minedBlockObservers,
         finalState.getNodeKey(),
-        finalState.getMessageFactory(),
-        finalState.getTransmitter(),
+        messageFactory,
+        new IbftMessageTransmitter(messageFactory, finalState.getValidatorMulticaster()),
```

Comment:
nit: extracting this a local var would this a bit cleaner

Reply:
done